codesign/README.md

```diff
@@ -38,16 +38,19 @@ cipd create -in build                   \
 
 ## How to use
 
-`codesign` is the executable binary, and can be called
+`codesign` is the executable binary, and can be called via
 
-```bash
-/path/to/codesign --commit <commit_sha> (--production)
---filepath <darwin-x64/FlutterMacOS.framework.zip>
---filepath <ios/artifacts.zip>
---filepath <more_file_path>
-```
+ ```bash
+ dart run bin/codesign.dart --[no-]dryrun
+ --codesign-cert-name="FLUTTER.IO LLC"
+ --codesign-team-id-file-path=/a/b/c.txt
+ --codesign-appstore-id-file-path=/a/b/b.txt
+ --app-specific-password-file-path=/a/b/a.txt
+ --input-zip-file-path=/a/input.zip
+ --output-zip-file-path=/b/output.zip
+ ```
 
-Use `/path/to/codesign --help` to learn more.
+The meaning and usage of each field is explained in detail
+in the comment section of `bin/codesign.dart`.
 
-**Note**: Do not add the --production flag unless the binaries are
-intended to be uploaded back to Google Cloud Storage.
\ No newline at end of file
+Use `/path/to/codesign --help` to learn more.
```

Comment:
```suggestion
Use `codesign --help` to learn more.
```

Once you've referenced how to run something, you can use a shorthand of it. This is especially true as users may run this via AOT or JIT

Reply:
sounds good, TIL about ahead of time and just in time compilation!